data/remnant missions.txt

```diff
@@ -493,5 +597,772 @@ mission "Remnant: Technology Available"
 				`	"I'm honored. Thank you."`
 					decline
 				`	"Is there anything else I can do to assist your people?"`
-			`	"Not that I am aware of right now," he says, "but we will call on you if a need ever arises."`
+			`	"There might be," he says, "Check the spaceports. If anyone has a job to be done, they will approach you. And there is almost always a few Korath ships that seem bent on destruction instead of loot, and those need to be dealt with."`
+				decline
+
+
+mission "Remnant: Heavy Laser"
+	name "Retrieve Heavy Lasers for the Remnant"
+	description "An engineer on <planet>, a Remnant world, has offered to pay you one million credits in exchange for obtaining two Heavy Lasers."
+	source
+		government Remnant
+	to offer
+		has "license: Remnant"
+		random < 75
+	on offer
+		require "Heavy Laser"
+		conversation
+			`As you are browsing the spacesport shops, a woman approaches you and introduces himself as a researcher at one of their weapons facilities. She pauses before beginning to recite a quick tune: "We can see that your ship has weapons that match human techniques, but are unlike anything we have record of. What kind of weapons are they?"`
+			choice
+				`	"They are Heavy Lasers."`
+					goto HeavyLasers
+				`	"I'd rather not share that."`
+					decline
+			label HeavyLasers
+			`	"Really? Our records indicate that humanity had basic laser weapons when we left, but nothing that would be considered useful modern weapons." She pauses to look down at his scanner. "And yet our scans indicate that they are capable of significant power output." `
+			`	Based on these scans, we may well be able to learn quite a bit from these weapons. Would you be able to acquire a pair of them for us? We could offer you a really good price for them."`
+			choice
+				`	"Sure, I would be glad to accept that deal."`
+				`	"Sorry, I'm really not interested in helping Remnant research."`
+					decline
+
+			`	After a bit of haggling, he agrees to pay you one and a half million credits in exchange for a cargo of two "Heavy Lasers."`
+				accept
+	on complete
+		outfit "Heavy Laser" -2
+		payment 1500000
+		conversation
+			`As <ship> descends towards its assigned landing pad, you notice the weapons researcher mounting some kind of weapon on a platform aimed down a firing range. Jumping down from the stand, she pushes a button and a flash of light erupts from the muzzle, and the target downrange is cut in half. She flips a leaver and raises the barrel, and people rush out onto the range with tools and devices to examine the destruction.`
```

Comment:
It will autocomplete. Whether or not it makes sense depends on how one interprets "As <ship> settles onto the pad closest to the research facility,". If one interprets it as moving to a pad closer to the facility for ease of unloading, it makes sense. If one interprets it as landing from a space voyage, not so much.